ParabolInc/parabol
# feat: add notifications for mention in reflections and show kudos preview

<img width="411" alt="image" src="https://github.com/ParabolInc/parabol/assets/466991/f622dc47-3f13-4964-a56e-eafd308d539f">

Fixes #9328

**In this PR:**
- I tried to make a generic `Mention`, because many similar things are duplicated every time. Let me know what you think.
- Added notification for mention in reflection
- Added reflection preview in the notification

**Not included:** 
- slack and email notification (Slack added here: #9377)

**How to test:**
- Start a retro in a team with few users
- Invite some users from the team to a retro, but not all users
- Add some reflections and mention various users in these reflections
- See users receive their mention notifications at the end of retro
- They also receive toasts
- See that mention notifications receive only users that were not in the retro
- See notifications include reflection preview
- Try same with non-anonymous retro reflections
- Try all above with kudos in reflections
- See kudos notifications now have reflection previews
